app/views/options-on-interstitial/check-state-pension-interstitial-1.html

```diff
@@ -0,0 +1,41 @@
+{% extends "layout.html" %}
+
+{% block page_title %}
+  {{ serviceName }}
+{% endblock %}
+
+{% block content %}
+
+<main id="content" role="main">
+  <div class="grid-row">
+    <div class="column-two-thirds">
+      <h1 class="heading-xlarge">
+        How do you want to sign in?
+      </h1>
+      <p>
+        Lorem ipsum dolor sit amet, te vix nisl splendide signiferumque, mel reque appetere cu. Pro eu populo conceptam. Id copiosae dissentias eam. Eum ea dico exerci dicunt, mea te habeo.
+      </p>
+    </div>
+  </div>
+  <form role="form" action="/check-state-pension-options-1" method="post" id="login-options">
+    <div class="form-group">
+      <fieldset>
+        <div class="multiple-choice">
+          <input id="government-gateway" type="radio" name="loginoption" value="government-gateway"/>
```

Comment:
At the moment when you interact with these radios, it's not clear what they're for.

To give context we need to move the heading so it's within the `fieldset` and within a `legend` this means when a user of assistive tech interacts with this form, the heading will be read out giving context to what they're doing.

Reply:
I've had a go at addressing this - can you take a look to see if it's what you were expecting?